resources/build/conanfile.py

```diff
@@ -41,7 +41,7 @@ def requirements(self):
             self.requires("ncurses/6.2@#54f100a8e4a94700d4123ed31420506c")
         # Same as ASWF CY2022 Docker image:
         # https://github.com/AcademySoftwareFoundation/aswf-docker/blob/master/ci-base/README.md
```

Comment:
[fix] This comment is also incorrect with the version update so needs removing.

Reply:
Done, thanks.